<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CRemoveLinesContainingWidget</class>
 <widget class="QWidget" name="CRemoveLinesContainingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remove Lines Containing... </string>
  </property>
  <widget class="QLabel" name="m_pLabelSearchLinesFor">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Search lines for:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_pLineEditSearchLinesFor">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>591</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnExtractColumns">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>50</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>EXTRACT COLUMN</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_pLabelAdd">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnAnd">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>AND</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnOr">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>OR</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_pLabelSearchField">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>searcg field.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnReset">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>RESET</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_pCheckBoxEnableRegExp">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>221</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Enable regular expression search. </string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnRemoveLinesContaining">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>REMOVE LINES CONTAINING...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnNotContaining">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>NOT CONTAINING...</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_pCheckBoxCaseSensitive">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>140</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Case sensitive</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_pCheckBoxDisplayRemoved">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>140</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Display removed</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnSelAllOutputText">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>140</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnClearAllOutputText">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>140</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="m_pTextEditInput">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>771</width>
     <height>161</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QTextEdit" name="m_pTextEditRemoved">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>771</width>
     <height>120</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnBrowse">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>520</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnSaveAs">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>520</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>SAVE AS</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="m_pRadioBtnDos">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>520</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Dos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnLoadFile">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>LOAD FILE</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_pLabelSelectedFile">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>520</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_pLineEditSaveAs">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>520</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QRadioButton" name="m_pRadioBtnUnix">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>520</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Unix</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_pCheckBoxDisplayOutput">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>140</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 127);
background-color: rgb(85, 255, 0);</string>
   </property>
   <property name="text">
    <string>Display output</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
